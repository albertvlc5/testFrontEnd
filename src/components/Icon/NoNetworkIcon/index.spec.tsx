import React from 'react';
import renderer from 'react-test-renderer';
import { NoNetworkIcon } from './';

describe('NoNetworkIcon', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NoNetworkIcon />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
