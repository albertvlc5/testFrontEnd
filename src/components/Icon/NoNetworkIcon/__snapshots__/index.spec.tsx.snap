// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoNetworkIcon renders correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight={140}
  bbWidth={140}
  fill="none"
  focusable={false}
  height={140}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      undefined,
      Object {
        "opacity": 1,
      },
      Object {
        "flex": 0,
        "height": 140,
        "width": 140,
      },
    ]
  }
  vbHeight={140}
  vbWidth={140}
  width={140}
>
  <RNSVGGroup
    fill={null}
    fillOpacity={1}
    fillRule={1}
    font={Object {}}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "fill",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  >
    <RNSVGPath
      d="M29.1665 73.2083C40.6965 63.6047 55.2275 58.3457 70.2332 58.3457C85.2388 58.3457 99.7699 63.6047 111.3 73.2083"
      fill={
        Array [
          0,
          4278190080,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "stroke",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={
        Array [
          0,
          4292927712,
        ]
      }
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    />
    <RNSVGPath
      d="M8.2832 52.5C25.3307 37.4731 47.2749 29.1821 69.9999 29.1821C92.7248 29.1821 114.669 37.4731 131.717 52.5"
      fill={
        Array [
          0,
          4278190080,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "stroke",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={
        Array [
          0,
          4292927712,
        ]
      }
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    />
    <RNSVGPath
      d="M49.7583 93.975C55.6803 89.7677 62.7647 87.5073 70.0291 87.5073C77.2935 87.5073 84.378 89.7677 90.3 93.975"
      fill={
        Array [
          0,
          4278190080,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "stroke",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={
        Array [
          0,
          4292927712,
        ]
      }
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    />
    <RNSVGPath
      d="M72.7929 108.293C73.1834 107.902 73.8166 107.902 74.2071 108.293C74.5976 108.683 74.5976 109.317 74.2071 109.707L70.9142 113L74.2071 116.293C74.5976 116.683 74.5976 117.317 74.2071 117.707C73.8166 118.098 73.1834 118.098 72.7929 117.707L69.5 114.414L66.2071 117.707C65.8166 118.098 65.1834 118.098 64.7929 117.707C64.4024 117.317 64.4024 116.683 64.7929 116.293L68.0858 113L64.7929 109.707C64.4024 109.317 64.4024 108.683 64.7929 108.293C65.1834 107.902 65.8166 107.902 66.2071 108.293L69.5 111.586L72.7929 108.293Z"
      fill={
        Array [
          0,
          4292927712,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={
        Array [
          "fill",
        ]
      }
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
